Update 04-14-april-cumulative-update.md (#53352)
dotnet/docs: docs/framework/release-notes/2026/04-14-april-cumulative-update.md

> # April 2026 cumulative update

Reviewer: ms.date should be update on line 4 as well

## Known issues in this release

> This update may fail to install when (applied through DISM)[https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/dism---deployment-image-servicing-and-management-technical-reference-for-windows?view=windows-11] to an offline Windows 10 IoT Enterprise LTSC 21H2 image. We are working on a resolution, and a fix is planned for inclusion in an upcoming .NET Framework update.

Reviewer 2: ```suggestion
This update might fail to install when [applied through DISM](/windows-hardware/manufacture/desktop/dism---deployment-image-servicing-and-management-technical-reference-for-windows?view=windows-11) to an offline Windows 10 IoT Enterprise LTSC 21H2 image. A fix for this issue will be included in an upcoming .NET Framework update.
```
